react-ui/src/components/Countdown.jsx

```diff
@@ -1,3 +1,8 @@
+/**
+ * Time remaining to submit or vote
+ * Displayed in header and entry description drawer
```

Comment:
Also now displayed on the submission page

Reply:
Edited